### PR TITLE
HPCC-14446 DOCS:Add WsECL and ECLWatch to Config book

### PR DESCRIPTION
**Pull request by g-pan**
Fix HPCC-14446 DOCS:Add WsECL and ECLWatch to Config book
Add sections to Config Manager doc to detail the tab and attributes
of the ESP Services, WsECL and ECL Watch.

Signed-off-by: G Panagiotatos greg.panagiotatos@lexisnexis.com
@JamesDeFabia please review
@garonsky please review
@xwang2713 please review

### CODE REVIEW

**Comment by JamesDeFabia** [comment]
Looks good from my perspective.

File: docs/XMLGeneration/CMakeLists.txt [hunk 1]
```diff
@@ -24,7 +24,7 @@ FILE(GLOB XSD_TEMPLATE_FILES RELATIVE "${XSD_SOURCE_DIR}" "${XSD_SOURCE_DIR}/*.x
 FILE(GLOB XSD_FILES RELATIVE "${XSD_SOURCE_DIR}" "${XSD_SOURCE_DIR}/*.xsd")
 
 # Skip some files which cause problem
-LIST(REMOVE_ITEM XSD_FILES "esp_service_wsecl2.xsd")
+##LIST(REMOVE_ITEM XSD_FILES "esp_service_wsecl2.xsd")
```
Comment (garonsky) [inline docs/XMLGeneration/CMakeLists.txt]: This line can just be removed.

File: initfiles/componentfiles/configxml/esp_service_wsecl.xsd [hunk 2]
```diff
@@ -25,6 +25,12 @@
             <xs:attributeGroup ref="EclServer"/>
             <xs:attributeGroup ref="Roxie"/>
             <xs:attributeGroup ref="Logging"/>
+         <!--DocAutoBuildCode-->
```
Comment (garonsky) [inline initfiles/componentfiles/configxml/esp_service_wsecl.xsd]: spacing

File: initfiles/componentfiles/configxml/esp_service_wsecl.xsd [hunk 3]
```diff
@@ -137,6 +145,12 @@
         </xs:attribute>
     </xs:attributeGroup>
     <xs:attributeGroup name="Roxie">
+        <!--DocAutoBuildCode-->
+        <xs:annotation>
+	     <xs:appinfo>
+	        <docid>MyWS_TbL04</docid>
+	     </xs:appinfo>
+         </xs:annotation>
```
Comment (garonsky) [inline initfiles/componentfiles/configxml/esp_service_wsecl.xsd]: spacing

File: initfiles/componentfiles/configxml/esp_service_wsecl2.xsd [hunk 4]
```diff
@@ -30,7 +30,13 @@
...
                 </xs:annotation>
                 <xs:complexType>
-                  <xs:attribute name="name" type="xs:string" use="required" default="">
+                        <!--DocAutoBuildCode-->
+                            <xs:annotation>
+		                <xs:appinfo>
+		                    <docid>MyWS2-T01</docid>
+		                </xs:appinfo>
+		              </xs:annotation>
+           <xs:attribute name="name" type="xs:string" use="required" default="">
```
Comment (garonsky) [inline initfiles/componentfiles/configxml/esp_service_wsecl2.xsd]: spacing

**Comment by garonsky** [comment]
Just some spacing issue. I didn't mark all of them.  Otherwise looks good.

**Pull request merged**